modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/maven/AbstractMavenClassLoaderModelLoader.java

```diff
@@ -112,46 +109,65 @@ public final ClassLoaderModel load(File artifactFile, Map<String, Object> attrib
     } else {
       return createLightPackageClassLoaderModel(artifactFile, attributes, artifactType);
     }
   }
 
-  private ClassLoaderModel createHeavyPackageClassLoaderModel(File artifactFile,
+  private ClassLoaderModel createHeavyPackageClassLoaderModel(File artifactFile, File classLoaderModelDescriptor,
                                                               Map<String, Object> attributes) {
-    File classLoaderModelDescriptor = getClassLoaderModelDescriptor(artifactFile);
+    return createHeavyPackageClassLoaderModel(artifactFile, classLoaderModelDescriptor, attributes,
+                                              of(getDeployableArtifactRepositoryFolder(artifactFile)));
+  }
 
+  protected ClassLoaderModel createHeavyPackageClassLoaderModel(File artifactFile, File classLoaderModelDescriptor,
+                                                                Map<String, Object> attributes,
+                                                                Optional<File> deployableArtifactRepositoryFolder) {
     org.mule.tools.api.classloader.model.ClassLoaderModel packagerClassLoaderModel =
         getPackagerClassLoaderModel(classLoaderModelDescriptor);
 
-    File deployableArtifactRepositoryFolder = getDeployableArtifactRepositoryFolder(artifactFile);
-
     final ArtifactClassLoaderModelBuilder classLoaderModelBuilder =
         newHeavyWeightClassLoaderModelBuilder(artifactFile, (BundleDescriptor) attributes.get(BundleDescriptor.class.getName()),
                                               packagerClassLoaderModel, attributes);
     classLoaderModelBuilder
-        .exportingPackages(new HashSet<>(getAttribute(attributes, EXPORTED_PACKAGES)))
-        .exportingPrivilegedPackages(new HashSet<>(getAttribute(attributes, PRIVILEGED_EXPORTED_PACKAGES)),
-                                     new HashSet<>(getAttribute(attributes, PRIVILEGED_ARTIFACTS_IDS)))
-        .exportingResources(new HashSet<>(getAttribute(attributes, EXPORTED_RESOURCES)));
-
-    Set<BundleDependency> patchBundleDependencies =
-        getPatchedBundledDependencies(artifactFile, deployableArtifactRepositoryFolder);
-
-    Set<BundleDependency> bundleDependencies =
-        packagerClassLoaderModel.getDependencies().stream().map(artifact -> {
-          Optional<BundleDependency> patchedBundledDependency =
-              patchBundleDependencies.stream().filter(bundleDependency -> bundleDependency.getDescriptor().getGroupId()
-                  .equals(artifact.getArtifactCoordinates().getGroupId()) &&
-                  bundleDependency.getDescriptor().getArtifactId().equals(artifact.getArtifactCoordinates().getArtifactId()))
-                  .findAny();
-          return patchedBundledDependency
-              .orElse(createBundleDependencyFromPackagerDependency(deployableArtifactRepositoryFolder).apply(artifact));
-        }).collect(toSet());
+        .exportingPackages(new HashSet<>(
```

Comment:
was this formatted?

Reply:
yes, I can remove that new line but formatter was "ok"